bin/flutterflow_cli.dart

```diff
@@ -142,6 +159,33 @@ Future _exportCode({
   }
 }
 
+// Extract files to the specified directory without a project-named
+// parent folder.
+void extractArchiveToCurrentDirectory(
+  Archive projectFolder,
+  String destinationPath,
+) {
+  for (final file in projectFolder.files) {
+    if (file.isFile) {
+      final data = file.content as List<int>;
+      final filename = file.name;
+
+      // Remove the `<project>/` prefix from path names.
+      final parentFolderIndex = filename[0] == '/'
+          ? (filename.substring(1)).indexOf('/')
+          : filename.indexOf('/');
+      final subStrDirectory = parentFolderIndex == -1
+          ? filename
+          : filename.substring(parentFolderIndex + 1);
+      final path = path_util.join(destinationPath, subStrDirectory);
```

Comment:
I'm not sure if this will work on Windows which uses backslash as a path separator. Perhaps it's safer to use `path_util` which knows how to handle these subtleties:

`final path = path_util.join(destinationPath, path_util.joinAll(path_util.split(filename).sublist(1)));`

Reply:
Good call, thanks!